distribution/lib/Standard/Base/0.0.0-dev/docs/api/Logging.md

```diff
@@ -6,4 +6,9 @@
     - Info
     - Severe
     - Warning
+- type Progress
+    - advance self amount:Standard.Base.Data.Numbers.Integer= -> Standard.Base.Logging.Progress
+    - log self detail:Standard.Base.Data.Text.Text -> Standard.Base.Logging.Progress
+    - run label:Standard.Base.Data.Text.Text up_to:Standard.Base.Data.Numbers.Integer action:Standard.Base.Any.Any -> Standard.Base.Any.Any
+    - to_text self -> Standard.Base.Any.Any
```

Comment:
The return type isn't correct. But the return type isn't correct in `Any.to_text` either. We might specify it by:
```
to_text self -> Text = ...
```
but it shows another problem: we aren't enforcing the arguments (their type, their count, their names) on overriden methods at all. Honestly, I don't even know how that was supposed to work in the mind of Enso language designers...